src/main/kotlin/build/buf/intellij/vendor/jetbrains/BufProtoFileResolver.kt

```diff
@@ -45,15 +42,14 @@ class BufProtoFileResolver : FileResolveProvider {
     override fun findFile(path: String, project: Project): VirtualFile? = findFiles(path, project).firstOrNull()
 
     private fun findFiles(path: String, project: Project): Sequence<VirtualFile> {
-        val files = LinkedHashSet<VirtualFile>()
-        for (bufConfig in FilenameIndex.getVirtualFilesByName(BufConfig.BUF_YAML, ProjectScope.getProjectScope(project))) {
-            val bufRoot = bufConfig.parent ?: continue
-            files.add(bufRoot.findFileByRelativePath(path) ?: continue)
+        val files = linkedSetOf<VirtualFile>()
+        val fs = VirtualFileManager.getInstance()
+        for (bufModuleConfig in BufIndexes.getProjectBufModuleConfigs(project)) {
```

Comment:
This is the main change - previously we'd assume that all paths were relative to the same directory where the `buf.yaml` is found. This change updates it to handle v2 module path resolution.